dockerfile

```diff
@@ -1,11 +1,20 @@
 FROM golang:1.20.1-alpine3.17 AS builder
-WORKDIR /tmp
-COPY . .
-RUN apk add make
-RUN make
 
-FROM alpine:latest
+COPY . /build
+
+WORKDIR /build
+
+ENV GOOS=linux
+ENV CGO_ENABLED=0
+
+RUN go get -d -v ./...
```

Comment:
I prefer to keep this stuff behind makefile targets so I only have to change one file. Those flags and the go get are good additions tho

Reply:
Rev'd back to using `make`, but it _did_ also requite a `git` prereq. Not a big deal since we're doing multi-stage, but weird.